Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/StartRide.hs

```diff
@@ -87,7 +95,21 @@ buildStartRideHandle merchantId = do
       }
 
 driverStartRide ::
-  (MonadThrow m, Log m, EsqLocDBFlow m r, MonadTime m, CoreMetrics m, MonadReader r m, HasField "enableAPILatencyLogging" r Bool, HasField "enableAPIPrometheusMetricLogging" r Bool) =>
+  ( MonadThrow m,
```

Comment:
the same, can we add function to `ServiceHandle` instead of changing constraint?

Reply:
working on adding the function in the service handle for both start ride and endride